examples/README.md

```diff
@@ -0,0 +1,32 @@
+# Examples
+
+This directory contains a set of examples utilising various d3fc features.
+
+## Viewing the examples
+
+Just load up one of the `index.html` files within an example directory.
```

Comment:
I misread this as "from within an example directory" - might be worth clarifying ...

"Just load up one of the example index.html files from an example directory, served from the project root"

Reply:
There's currently no need to run up a web server, you can literally just open the file. I was expecting we'd add some more detailed instructions or a package script if/when that is required.

Reply:
I've reworded it to hopefully make that clear

Reply:
> There's currently no need to run up a web server, you can literally just open the file
Oh - that's cool, you should definitely make that clear. I assumed a dev server was required!